### PR TITLE
Finished chapter WiFi HotSpot (fixes #140)

### LINKED ISSUE #140: Porting WiFi Hotspot
Porting started.

### PR DESCRIPTION
**Pull request by StephanStS**
Fixes issue #140.

### CODE REVIEW

File: docs/software/wifi_hotspot.md [hunk 1]
```diff
@@ -2,12 +2,80 @@
...
+
+    - 1x Ethernet connection
+    - 1x Supported USB WiFi adapter or onboard WiFi. This may vary depending on device and available WiFi drivers/modules. However, common adapters (e.g.: Realtek) should be fine.
+
+=== "Initial connection credentials"
+
+    Use the following credentials to initially connect devices to your hotspot.
+
+    - SSID = *DietPi-HotSpot*
+    - Access Key = *dietpihotspot*
```
Comment (MichaIng) [inline docs/software/wifi_hotspot.md:44]: What do you think about wrapping all such values into code tags? `DietPi-HotSpot` and `dietpihotspot`, so they don't need to be added to the wordlist. So basically making everything to inline-code what is meant to be copy&pasted to console or input field.

File: docs/software/wifi_hotspot.md [hunk 2]
```diff
@@ -2,12 +2,80 @@
...
+- [WiFi HotSpot](#wifi-hotspot_1)
+
+![DietPi WiFi hotspot tor](../assets/images/dietpi-software-wifihotspot-torhotspot.png){: style="width:550px"}
+
+=== "Requirements"
+
+    The requirements are:
+
+    - 1x Ethernet connection
+    - 1x Supported USB WiFi adapter or onboard WiFi. This may vary depending on device and available WiFi drivers/modules. However, common adapters (e.g.: Realtek) should be fine.
```
Comment (MichaIng) [inline docs/software/wifi_hotspot.md:70]: Let's not mention `Realtek` here explicitly, better another brand, `Atheros` or something. Realtek explicitly has a set of WiFi chips which do NOT work with the native `hostapd` software. Explained here: https://github.com/pritambaral/hostapd-rtl871xdrv

On RPi, the firmware repository ships a `hostapd` package which has this support included and Armbian ships a dedicated `hostapd-realtek` package, on other ARM platforms we ship an own compiled package: https://dietpi.com/downloads/binaries/all/hostapd_2.5_all.zip
But that is all moreless workarounds, often outdated versions, and not available for x86.

**Review by MichaIng — CHANGES_REQUESTED** [review]

File: docs/software/wifi_hotspot.md (hunk 2 above)
Comment (StephanStS) [reply docs/software/wifi_hotspot.md:70]: Done.

File: docs/software/wifi_hotspot.md (hunk 1 above)
Comment (StephanStS) [reply docs/software/wifi_hotspot.md:44]: Ok, done.  
I think I have to change this on more documents. I will check.

**Pull request merged**